spec/models/post_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it 'is valid with a title' do
+    user = User.create(name: 'Newton', posts_counter: 0)
+    post = user.posts.new(title: 'Hello')
+    expect(post).to be_valid
+  end
```

Comment:
- [ ] According to the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md) please add unit specs for your models' method `def recent_comments` in `post.rb`🛠🤗